src/Equinor.ProCoSys.IPO.Domain/AggregateModels/InvitationAggregate/IInvitationRepository.cs

```diff
@@ -8,14 +7,14 @@ namespace Equinor.ProCoSys.IPO.Domain.AggregateModels.InvitationAggregate
 {
     public interface IInvitationRepository : IRepository<Invitation>
     {
+        public Project GetProject(string projectName);
         void UpdateProjectOnInvitations(string projectName, string description);
-        void UpdateCommPkgOnInvitations(string projectName, string commPkgNo, string description);
+        void UpdateCommPkgOnInvitations(string proCoSysGuid, string commPkgNo, string description, Project project);
         void UpdateMcPkgOnInvitations(string projectName, string mcPkgNo, string description);
         void UpdateFunctionalRoleCodesOnInvitations(string plant, string functionalRoleCodeOld, string functionalRoleCodeNew);
         void RemoveParticipant(Participant participant);
         void RemoveAttachment(Attachment attachment);
         void RemoveInvitation(Invitation invitation);
-        void MoveCommPkg(string fromProject, string toProject, string commPkgNo, string description);
         void MoveMcPkg(string projectName, string fromCommPkgNo, string toCommPkgNo, string fromMcPkgNo, string toMcPkgNo, string description);
```

Comment:
remove since it's now used in tests only. Remember to remove implementation also